src/components/ReplyField/ReplyField.tsx

```diff
@@ -58,6 +58,10 @@ export default class ReplyField extends React.Component<Props, State> {
         fetchCollaborators(trimmedQuery);
     }, DEFAULT_COLLAB_DEBOUNCE);
 
+    componentDidMount(): void {
+        window.addEventListener('resize', this.updatePopupReference);
```

Comment:
Should this be debounced? Does this conflict with preview's resize handling at all?

Reply:
I don't think debounce is needed here since we don't need to catch the new dimensions. We just trigger a popper update, it shouldn't conflict with preview's resize handling.

Reply:
Hm, it looks like calling `this.updatePopupReference` triggers a `setState` and also a bunch of logic to get data from the DOM via `getVirtualElement`. How can we come to further certainty regarding "shouldn't" on the above?

Reply:
One possible conflict that might crop up could be caused by the Preview SDK's own debounced resize handler. Is it possible that the following re-render in the Preview SDK could occur _after_ the resize handler in Annotations? Would that cause the popup to be mispositioned? There may be other examples, such as resize handlers in BUIE.

Reply:
Discussed offline.